src/util.ts

```diff
@@ -91,7 +91,7 @@ async function getBase(
 
     return contents;
   } catch (err) {
-    throw new Error(`${filePath} Not Found`);
+    throw new Error(`Error: ${filePath}: ${err}`);
```

Comment:
So good catch - how about instead, we prepend the `${filePath} not found` to the original `err.message`, and then re-throw `err` so we maintain the full stack trace?

Reply:
could you instead capture the stack trace, so that the original stack is in the error, but throw the new message? it feels a little weird mucking with an error (a little too magic).